scripts/webhook-deploy.sh

```diff
@@ -54,6 +62,8 @@ done
 [ -z ${kubectl} ] && kubectl="kubectl"
 [ -z ${mode} ] && mode="preprogrammed"
 [ -z ${namespace} ] && namespace="default"
+[ -z ${uid} ] && uid="$(id -u)"
```

Comment:
would some standard non-login user be better default? or print a warning if uid/gid are not set?

Reply:
done